ui/src/app/pages/workspace/workspace-library.tsx

```diff
@@ -60,6 +60,9 @@ const libraryTabs = {
   },
   PHENOTYPE_LIBRARY: {
     title: 'Phenotype Library',
+    description: <div>These workspaces demonstrate how computable electronic phenotypes can be implemented within
+      the <i>All of Us</i> data using examples of previously published phenotype algorithms. You can open the
```

Comment:
"with All of Us data **set**"

or maybe

"can be implemented with All of Us data"